src/main.ipynb

```diff
@@ -52,9 +53,23 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 7,
    "metadata": {},
-   "outputs": [],
+   "outputs": [
+    {
+     "data": {
+      "text/plain": "Accordion(children=(UploadFileWidget(children=(FileChooser(path='C:\\Users\\alext\\PycharmProjects\\gam-ml-test\\db…",
+      "application/vnd.jupyter.widget-view+json": {
+       "version_major": 2,
+       "version_minor": 0,
+       "model_id": "2d8daa9dfd014861b4b7216049c13d3e"
+      }
+     },
+     "execution_count": 7,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
```

Comment:
`*.ipynb` metadata changes are "junk" changes for VCS, I'm working on automatic pre-comit tool for clearing all outputs and execution counts, but for now you should do it manually and not add them to commits.